README.md

```diff
@@ -1,40 +1,72 @@
 
-To learn more about Habitat, please visit the [Habitat website](https://www.habitat.sh).
+See a quick demo of how to build, deploy and manage an application with Habitat:
 
-Participation in the Habitat community is governed by the [code of conduct](https://github.com/habitat-sh/habitat/blob/master/CODE_OF_CONDUCT.md).
+[![Build, Deploy and Manage with Habitat (5:33)](images/overview-youtube-image.jpg)](http://www.youtube.com/watch?v=VW1DwDezlqM)
 
 # Table of Contents
+* [Infographics](#infographics)
+* [Hands-on Demos](#hands-on-demos)
 * [Install](#install)
 * [Contribute](#contribute)
 * [Documentation](#documentation)
-* [Code Organization](#repo-organization)
+* [Code Organization](#code-organization)
 * [Roadmap](#roadmap)
 * [Community and support](#community-and-support)
 * [Building](#building)
 * [Further reference material](#further-reference-material)
+* [Code of Conduct](#code-of-conduct)
 * [License](#license)
 
+## Infographics
+Graphics that will help you and your team better understand the concepts and how they fit together into the larger Habitat ecosystem.
+### Where Habitat Fits
+
+[![Habitat Flow Infographic](images/habitat-flow-infographic.png)](http://habitat.sh)
+
+[//]: # (Try the interactive infographics on the [website](http://habitat.sh)
+
+### How Habitat Works
+* [Architecture Overview](https://www.habitat.sh/images/infographics/habitat-architecture-overview-7e40c4b3.png)
+* [**Initial Package Build** Flow](https://www.habitat.sh/images/infographics/habitat-initial-package-build-flow-43ccb287.png)
+* [**Application Rebuild** Flow](https://www.habitat.sh/images/infographics/habitat-application-rebuild-flow-10c48cfb.png)
+* [**Dependency Update** Flow](https://www.habitat.sh/images/infographics/habitat-dependency-update-flow-f4895875.png
+)
+* [Promote Packages Through **Channels**](https://www.habitat.sh/images/infographics/habitat-promote-packages-through-channels-06506fd0.png)
+
+### Habitat and **Docker**
+* [Initial Docker Container Publishing Flow](https://www.habitat.sh/images/infographics/habitat-initial-docker-container-publishing-flow-1f2ba489.png)
+* [Automated Docker Container Publishing Flow](https://www.habitat.sh/images/infographics/habitat-automated-docker-container-publishing-flow-3cdf9f91.png
+)
+
+### Habitat and **Kubernetes**
+* [Three Tiers of Service Deployment](https://www.habitat.sh/images/infographics/habitat-and-kubernetes-three-tiers-of-service-deployment-8c85b4ca.png)
+* [Deploy Services to Kubernetes with Habitat](https://www.habitat.sh/images/infographics/deploy-services-to-kubernetes-with-habitat-flow-567c4ce9.png)
+
+*View all infographics in [docs](https://www.habitat.sh/docs/reference/#infographics)*
```

Comment:
Super minor, but could we capitlize `Docs`?

Reply:
Yeah. Good catch!